### PR TITLE
Feat/blossom/store wire up

### PR DESCRIPTION
**Pull request by Blossom1-chike**
## Description
[Please put a description of the changes this branch introduces here.]
store wireup with backend

## To Test
[Please write instructions for testing your changes locally here. Include any data setup, tests, urls, and potential snags in local testing.]

## Checklist
- [ ] I have looked over the diffs.
- [ ] There are no errors/failing tests from my changes on this branch.
- [ ] I have requested a review on this PR and re-assigned the Jira ticket.

## Screenshots
[Any screenshots of the changes can go here.]

## Extra Notes
[Please include any extra notes you may have about the changes here.]

### CODE REVIEW

File: src/components/auth/forms/ResetPassword.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const ResetPassword = () => {
           email: value.email,
         }
       );
-
+console.log(success, responseData)
```
Comment (ObiTam) [inline src/components/auth/forms/ResetPassword.js:33]: remove console.log

File: src/components/profile/index.js [hunk 2]
```diff
@@ -45,9 +67,75 @@ const Profile = () => {
...
-    dispatch(updateProfile({ profile: { ...profileData } }));
-    setEditProfile(false);
+  const [currentTab, setCurrentTab] = useState("Profile");
+  const [editProfile, setEditProfile] = useState(false);
+
+  const handleProfileFormSubmit = async () => {
+    await updateStoreInfo();
+  };
+
+  const updateStoreInfo = async () => {
```
Comment (ObiTam) [inline src/components/profile/index.js:77]: we don't need this function. put it directly inot handle profile form submit

Comment (ObiTam) [reply src/components/profile/index.js:77]: there's no need for a sepereate button to upload an image. 

Store image update isn't even working. 

Do it this way instead. when an image is uploaded for an update you load it into data. 
when a user clicks save you first check to see if data is not null fo the image. if it is then you upload it and await the upload. if it doesn't work you set an error that we couldn't upload the image. if it works you then use that image id in your update store call to update the image.

**Review by ObiTam — CHANGES_REQUESTED** [review]
image
opening times is not wired up. discuss with samuel

Merge latest changes from main and deal with conflicts

Did you work on create store? for new admins?

File: src/components/profile/index.js (hunk 2 above)
Comment (Blossom1-chike) [reply src/components/profile/index.js]: done

File: src/components/auth/forms/ResetPassword.js (hunk 1 above)
Comment (Blossom1-chike) [reply src/components/auth/forms/ResetPassword.js]: done

File: src/utils/index.js [hunk 3]
```diff
@@ -28,12 +28,11 @@ export const handleAvatarSubmit = async (image, id) => {
     });
     return;
   }
-  toast.info(`⏳ uploading image.....`, { autoClose: 2000 });
   try {
     const files = new FormData();
     files.append("files", image);
     files.append("storeId", `${id}`);
-
+    toast.info(`⏳ uploading image.....`, { autoClose: 2000 });
```
[the remark this answers is not in the document]
Comment (ObiTam) [reply src/utils/index.js:35]: why are you throwing an error in the catch statememend

**Review by ObiTam — CHANGES_REQUESTED** [review]
the delete button doesn't work 

image
it should remove the image there but it shouldn't allow you to submit  without uploading a new image. 

if you're adding a toast for image is uploading which I don't think is necessary then you need to add an image upload successful and an image upload error. I'd rather there be one toast for successful store details upload and failed store details upload. 

Well done. just a little more.

**Comment by ObiTam** [comment]
> image opening times is not wired up. discuss with samuel
> 
> Merge latest changes from main and deal with conflicts
> 
> Did you work on create store? for new admins?

@Blossom1-chike  any update?

**Comment by Blossom1-chike** [comment]
> > image opening times is not wired up. discuss with samuel
> > Merge latest changes from main and deal with conflicts
> > Did you work on create store? for new admins?
> 
> @Blossom1-chike any update?

i told sameuel, he hasn't replied on slack

**Review by ObiTam — APPROVED** [review]
@Eshy10 Please review.

**Comment by Eshy10** [comment]
@Blossom1-chike i am a new store owner but i already see a profile picture with the option to delete it, it should show the default profile picture which is the placeholder.
Screenshot 2024-04-02 at 04 55 16

- when i try to add an holiday i get an error

Screenshot 2024-04-02 at 04 57 33

Screenshot 2024-04-02 at 04 57 47

**Comment by ObiTam** [comment]
Hi @Blossom1-chike Please resolve the conflicts then i'll take a look.

**Review by ObiTam — APPROVED** [review]

**Comment by Eshy10** [comment]
we are moving the fix i noticed to another pr @Blossom1-chike @ObiTam

**Pull request merged**